app/views/cards/show.html.erb

```diff
@@ -1,5 +1,52 @@
-<h1><%= @card.name %></h1>
-<h2><%= @card.user.name %> </h2>
-<%= render @events %>
-<%= paginate @events %>
-<%= render "events/form" %>
+<section class="row">
+  <%= link_to @project do %>
+    <button class="btn btn-danger"><%= t "card.close" %></button>
+  <% end %>
+  <article class="col-md-6 col-md-offset-3 card_detail">
+    <header class="card_title">
+      <h3>
+        <span class="edit"><%= @card.name %></span>
+      </h3>
+      <h4><%= t "card.in_list" %>
+        <span id="task_name"><%= @card.task.name %></span>
+        <%= form_tag @card, id: "task_form", method: :put do %>
+          <%= select_tag "task_id",
+            options_for_select(@project.tasks.collect{|u| [u.name, u.id]}
```

Comment:
Chuyển toàn bộ đoạn @project.tasks.collect{|u| [u.name, u.id]}
              .insert(0, t("card.select_task")) vào helper rồi dọi ra